src/main/java/flash/npcmod/client/gui/screen/dialogue/DialogueBuilderScreen.java

```diff
@@ -205,20 +194,8 @@ public boolean mouseClicked(double mouseX, double mouseY, int button) {
                         this.selectedNode = null;
                     }
                     if (button == 1) {
-                        this.createMenuWidget.setY(mouseY);
-                        this.createMenuWidget.setX(mouseX);
-                        this.createMenuWidget.setVisible(true);
-                    }
-                }
-                if (button != 1 && this.createMenuWidget.isVisible()) {
-                    this.createMenuWidget.clickedOn(mouseX, mouseY);
-                    if (!this.createMenuWidget.getSelectedOption().isEmpty()) {
-                        String option = this.createMenuWidget.getSelectedOption();
-                        if (option.equals("Create Dialogue")) {
-                            createNode(mouseX, mouseY);
-                        }
+                        createNode(mouseX, mouseY);
                     }
-                    this.createMenuWidget.setVisible(false);
                 }
                 // TODO make sure not to check during parent setting.
```

Comment:
This seems important

Reply:
Also, you can't edit the text and response of dialogues. The overlay shows up, but there's no input field.